roles/confluent.kafka_rest/templates/kafka-rest_kerberos_ssl_customcerts.properties.j2

```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
 {% include './includes/base_kafka-rest.properties.j2' %}
 
-client.bootstrap.servers={% for inventory_hostname in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+client.bootstrap.servers={% for inventory_hostname in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{kafka_port}}{% endfor %}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.